include/crocoddyl/core/integrator/rk4.hpp

```diff
@@ -123,24 +136,32 @@ struct IntegratedActionDataRK4Tpl : public ActionDataAbstractTpl<_Scalar> {
   virtual ~IntegratedActionDataRK4Tpl() {}
 
   VectorXs dx;
+  std::vector<VectorXs> u_diff;
   std::vector<boost::shared_ptr<DifferentialActionDataAbstractTpl<Scalar> > > differential;
```

Comment:
Renamed this to differentials (a plural word). Note that `differential` is already stored in the abstract class.

Reply:
Both `controls` and `differentials` should be `std::array`, but this a tangent discussion. (please do no worry about it)

Reply:
Ok for adding the "s", but note that `differential` is not stored in the abstract class.

Reply:
Then, I think we should include it in the base API. Do you have any point against to it?

Just let me know your feedback, I will handle it myself (as in any case I need to do some work on this class)

Reply:
The reason why I didn't include differential in the base class is that in the case of Euler you just need one differential, whereas for RK4 you need 4 of them. We could of course have a single differential in the base class, and then add a 3d vector of differentials in the RK4 class, but that would break the current implementation of the RK4 integration, which uses a for loop assuming the 4 differential data structures are stored in a vector.

Reply:
I renamed back to differential, despite I liked more the plural one. The reason is to avoid deprecate the RK4 code.